samples/connectedhomeip/lock/README.rst

```diff
@@ -20,8 +20,11 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf52840dk_nrf52840, nrf5340dk_nrf5340_cpuapp
 
-If you want to commission and control the lock device remotely :ref:`through a Thread network <chip_lock_sample_network_mode>`, use the `Android CHIPTool`_ application as the CHIP controller.
-You will need a smartphone compatible with Android for this purpose.
+For remote testing scenarios, you also need the following:
+
+* If you want to commission the lock device and :ref:`control it remotely <chip_lock_sample_network_mode>` through a Thread network:
+
+  * A smartphone compatible with Android for using the `Android CHIPTool`_ application as the CHIP controller.
 
```

Comment:
@Damian-Nordic , no Test mode for the lock app is intended, right?

Reply:
There is testing mode described in this document, but we don't have a dedicated `lock-controller` app, so there's nothing to mention about in the requirements section regarding the test mode.